src/components/Contracts/ContractItem/index.js

```diff
@@ -52,6 +53,23 @@ function ContractItem({
     parties,
     partiesSignaturesList,
   });
+  const signatories = useMemo(() => infoContract.filter(
+    ({ itemsOrItem }) => !Array.isArray(itemsOrItem) || itemsOrItem.length !== 0,
+  ).reduce((items, { itemsOrItem, name }) => {
```

Comment:
there already is a function for getting identities. use it. Its noderpccall file  getIdentities